dbschema/src/main/java/org/folio/rest/persist/ddlgen/Index.java

```diff
@@ -83,53 +85,40 @@ public String getFinalSqlExpression(String tableLoc) {
         if(i != 0) {
           result .append(" , ");
         }
-        appendExpandedTerm(tableLoc, splitIndex[i], result);
+        result.append(formatExpandedTerm(tableLoc, splitIndex[i]));
```

Comment:
Can you revert this back to `appendExpandedTerm(tableLoc, splitIndex[i], result);`?